lib/Cargo.toml

```diff
@@ -47,6 +49,9 @@ toml = "0.8.12"
 xshell = { version = "0.2", optional = true }
 uuid = { version = "1.7.0", features = ["v4"] }
 
+[build-dependencies]
+shadow-rs = { version = "0.27", default-features = false, features = ["git2"] }
```

Comment:
I'm confused, why does it need to be both a runtime and build dependency?  Isn't it just a build dependency?  Is that for the clap integration?  I guess this is an upstream question, as we're just following their recommendations here.

Reply:
Good question. I'm not completely sure, tbh